src/plugins/messenger/MessengerPreview/components/MessengerGenericTemplate/MessengerGenericTemplate.tsx

```diff
@@ -89,6 +90,8 @@ export const getMessengerGenericTemplate = ({
             const { image_url, image_alt_text, title, subtitle, buttons, default_action } = element;
 
             const isCentered = this.props.config.settings.designTemplate === 2;
+            const ariaLabelForMessengerTitle = default_action ? title + " .Opens in new tab" : title;
```

Comment:
`title` might be unset, could result in errors

Reply:
dot and space have to be switched

Reply:
fixed